config/database.yml

```diff
@@ -5,6 +5,7 @@ development: &default
   pool: 5
   username: postgres
   password: postgres
+  host: db
```

Comment:
This line might break the application when it's run outside of Docker.

A possible alternative would be passing in the database URL as an environment variable – suggested at http://guides.rubyonrails.org/configuring.html.